test/monitoring_console/monitoring_console_test.go

```diff
@@ -49,38 +49,131 @@ var _ = Describe("Monitoring Console test", func() {
 	})
 
 	Context("Deploy Monitoring Console", func() {
-		It("smoke, monitoring_console: can deploy MC CR", func() {
+		It("smoke, monitoring_console: can deploy MC CR adn", func() {
+			/*
+				Test Steps
+				1. Deploy Monitoring Console
+				2. Deploy Standalone
+				3. Wait for Monitoring Console status to go back to READY
+				4. Verify Standalone configured in Monitoring Console Config Map
+				5. Verify Monitoring Console Pod has correct peers in Peer List
+			*/
+
+			// Deploy MC and wait for MC to be READY
```

Comment:
nit: Isn't "wait for MC to be READY" part actually done on line 67?

Reply:
It supposed to be for the section from 62-67 however clearly its not clear I will modify